CONTRIBUTING.md

```diff
@@ -51,3 +117,13 @@ To develop for the `find-the-words` plugin, see the instructions in the [README]
     - If you want to run a specific test file, run `pytest tests/testfiles/{test file name}`
     - If you want to run a specific test case under the specific file, run `pytest tests/testfiles/{test file name}::{test function name}`
 
+## Development Tools
```

Comment:
Is it possible to move this work to this PR (https://github.com/gvwilson/mossball/pull/81)?